components/_patterns/01-atoms/03-text/01-heading/_heading.scss

```diff
@@ -35,7 +35,7 @@
 }
 
 .h1 {
-  @include heading($text-transform: uppercase);
+  @include heading($font-size: 2.688em, $color: $grey-5, $font-weight: 700, $letter-spacing: 0.028em);
```

Comment:
Please don't change these "default" .h1 styles. Other modifier styles that inherit it are not expecting the additional styles.